inst/validation/bbr-requirements.yaml

```diff
@@ -566,6 +566,10 @@ NMJ-R006:
   description: nm_join() warns on skipping table with wrong number of rows
   tests:
   - BBR-NMJ-006
+NMJ-R007:
+   description: nm_join() informatively errors if .join_col has duplicate values in any of the input tables
```

Comment:
Is the check and the error documented in `.Rd` file?  Could we also mention that duplicates in `$TABLE` outputs can be caused by lack of output width and that FORMAT might need to be stated by the user in the control stream to have sufficient width to avoid truncating `.join_col`.